.github/workflows/e2e.yaml

```diff
@@ -17,12 +17,17 @@ jobs:
         go-version: '~1.17'
     - name: Install kind
       run: |
-        curl -sLo kind "$(curl -sL https://api.github.com/repos/kubernetes-sigs/kind/releases/latest | jq -r '[.assets[] | select(.name == "kind-linux-amd64")] | first | .browser_download_url')"
+        curl -sLo kind "$(curl -sL https://api.github.com/repos/kubernetes-sigs/kind/releases/43733955 | jq -r '[.assets[] | select(.name == "kind-linux-amd64")] | first | .browser_download_url')"
```

Comment:
Can we avoid using a magic number (commit? psuedotag?) and instead use a environment variable to track the kind version we want to install:

```
- name: Install kind
  env:
    KIND_VERSION: v0.11.1
  run: |
    curl -sLo kind "$(curl -sL https://github.com/kubernetes-sigs/kind/releases/download/${KIND_VERSION} | jq ...)"
```